### PR TITLE
Added customizable serviceaccount

### PR DESCRIPTION
**Pull request by balazs92117**
Added customizable serviceaccount with default values of create it, but do not automount it.

### CODE REVIEW

**Comment by desaintmartin** [comment]
thanks! Here are a few notes.

File: charts/clamav/templates/statefulset.yaml [hunk 1]
```diff
@@ -25,7 +25,8 @@ spec:
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
       {{- end }}
-      automountServiceAccountToken: false
+      serviceAccountName: {{ include "clamav.serviceAccountName" . }}
+      automountServiceAccountToken: {{ .Values.serviceAccount.name }}
```
[the remark this answers is not in the document]
Comment (balazs92117) [reply charts/clamav/templates/statefulset.yaml:29]: Not necessary. It is possible to create an service account but not automount it. It is commonly used for security reasons.

[the remark this answers is not in the document]
Comment (balazs92117) [reply charts/clamav/templates/statefulset.yaml]: Now I see what you meant. Sorry, I inserted the wrong variable name.

**Review by desaintmartin — APPROVED** [review]

**Comment by desaintmartin** [comment]
Thanks for the contribution, and sorry for the delay!

**Pull request merged**